src/components/Footer.module.css

```diff
@@ -121,10 +122,10 @@
 }
 
 .topFooterContainer {
-  background-color: black;
+  background-color: rgb(0, 62, 82);
   display: flex;
   justify-content: space-between;
-  padding: 0.5%;
+  padding: 1em 5.5em;
```

Comment:
I feel like you should be more consistent with units on absolute sizes.  ie: either px or rem or em.